packages/gatsby-image/src/index.js

```diff
@@ -69,6 +75,55 @@ function getIO() {
   return io
 }
 
+function generateImageSources(imageVariants) {
+  return imageVariants.map(({ src, srcSet, srcSetWebp, media, sizes }) => (
+    <React.Fragment key={src}>
+      {srcSetWebp && (
+        <source
+          type="image/webp"
+          media={media}
+          srcSet={srcSetWebp}
+          sizes={sizes}
+        />
+      )}
+      <source media={media} srcSet={srcSet} sizes={sizes} />
+    </React.Fragment>
+  ))
+}
+
+function generateTracedSVGSources(imageVariants) {
+  return imageVariants.map(variant => (
```

Comment:
You destructor above so let's destructure here too.
```suggestion
  return imageVariants.map(({ src, media, tracedSVG })=> (
```